### PR TITLE
Structure for bottom nav bar

### PR DESCRIPTION
**Pull request by ayu-ayushisood**
Add a structure for Bottom navigation and add tabs.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -7,106 +8,25 @@
...
   const isDarkMode = useColorScheme() === 'dark';
 
   const backgroundStyle = {
     backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
   };
 
   return (
-    <SafeAreaView style={backgroundStyle}>
+    <SafeAreaView
+      style={[backgroundStyle, {backgroundColor: '#178be7', flex: 1}]}>
```
Comment (karanasthana) [inline App.js:25]: Let's make separate files for styles? Or use Stylesheets per file

File: ios/Podfile.lock [hunk 2]
```diff
@@ -272,6 +272,8 @@ PODS:
   - React-jsinspector (0.66.3)
```
Comment (karanasthana) [inline ios/Podfile.lock:272]: this file is not required to be pushed. Let's add it to gitnignore

File: App.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-native/no-inline-styles */
```
Comment (karanasthana) [inline App.js:1]: remove after moving all the styles to StyleSheet or separate styling files

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,9087 @@
+{
```
Comment (karanasthana) [inline package-lock.json:1]: need not be added to the repo

File: src/components/BottomNavigation.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react-native/no-inline-styles */
+import * as React from 'react';
+import {Text, View} from 'react-native';
+import {NavigationContainer} from '@react-navigation/native';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import FontistoIcons from 'react-native-vector-icons/Fontisto';
+
+const JavascriptComponent = () => {
```
Comment (karanasthana) [inline src/components/BottomNavigation.js:9]: Eventually, these will all just be different files, right?

File: src/components/BottomNavigation.js [hunk 6]
```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react-native/no-inline-styles */
```
Comment (karanasthana) [inline src/components/BottomNavigation.js:1]: Remove after all inline styles have been moved to the stylesheet or to a separate file

File: App.js (hunk 1 above)
Comment (ayu-ayushisood) [reply App.js:25]: yes, we'll do that eventually

File: package-lock.json (hunk 4 above)
Comment (ayu-ayushisood) [reply package-lock.json:1]: should be in gitignore

File: src/components/BottomNavigation.js (hunk 5 above)
Comment (ayu-ayushisood) [reply src/components/BottomNavigation.js:9]: correct

**Pull request merged**